conference/index.html

```diff
@@ -4,6 +4,7 @@
 id: conference
 ---
 
+<script src='https://js.tito.io/v1' async></script>
```

Comment:
Can confirm it doesn't complain on my local environment. I can remove it if you'd prefer ...but my git-fu isn't great so I'll probably just create a new PR